webpack/components/NewTemplateSync/components/NewTemplateSyncForm/NewTemplateSyncFormConstants.js

```diff
@@ -0,0 +1 @@
+export const formName = 'newTemplateSync';
```

Comment:
The convention for such a constant is `FORM_NAME`, consider changing to `NEW_TEMPLATE_SYNC_FORM_NAME`.